x-pack/legacy/plugins/ml/server/routes/system.ts

```diff
@@ -0,0 +1,204 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { schema } from '@kbn/config-schema';
+
+import { Request } from 'hapi';
+import { RequestHandlerContext } from 'kibana/server';
+import { wrapError } from '../client/error_wrapper';
+import { mlLog } from '../client/log';
+import { privilegesProvider } from '../lib/check_privileges';
+import { isSecurityDisabled } from '../lib/security_utils';
+import { spacesUtilsProvider } from '../lib/spaces_utils';
+import { licensePreRoutingFactory } from '../new_platform/licence_check_pre_routing_factory';
+import { RouteInitialization } from '../new_platform/plugin';
+
+export function systemRoutes({
+  router,
+  xpackMainPlugin,
+  spacesPlugin,
+  cloud,
+}: RouteInitialization) {
+  async function getNodeCount(context: RequestHandlerContext) {
+    const filterPath = 'nodes.*.attributes';
+    const resp = await context.core.elasticsearch.adminClient.callAsInternalUser('nodes.info', {
```

Comment:
`callAsInternalUser` is part of `mlClient` so this could be `context.ml!.mlClient.callAsInternalUser`

Reply:
Done in https://github.com/elastic/kibana/pull/57835/commits/e60e183aecb29f087a418637eea683ce7f84e3c7